core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/ServerConfig.java

```diff
@@ -18,6 +18,15 @@
 public interface ServerConfig {
     /**
      * Port used by server.
+     *
+     * <p>It may contain protocol(TCP or UDP) after '/' symbol.
+     * If protocol is missed TCP will be used by default.
+     * Example:
+     * <ul>
+     *     <li>8080/TCP</li>
```

Comment:
Matching is key-sensitive at least if you're using `ServersMapper`, so you should either fix javadoc or make the mapping case-insensitive